src/Jackett.Common/Definitions/piratbitl.yml

```diff
@@ -720,15 +725,37 @@ search:
     title:
       selector: td a.genmed
       filters:
+        - name: replace
+          args: ["Кураж-Бамбей", "kurazh"]
+        - name: replace
+          args: ["Кубик в Кубе", "Kubik"]
+        - name: replace
+          args: ["Кравец", "Kravec"]
+        - name: replace
+          args: ["Пифагор", "Pifagor"]
+        - name: replace
+          args: ["Невафильм", "Nevafilm"]
+        - name: replace
+          args: ["Лицензия", "Lic"]
+        - name: replace
+          args: ["селезень", "selezen"]
         # normalize to SXXEYY format
         - name: re_replace
-          args: ["[\\:\\-\\/\\|]", " "]
```

Comment:
Did you mean to remove this completely and only now replace ` / `? (same for piratbit obviously)